topics/basics/basic_action_system.md

```diff
@@ -41,6 +41,18 @@ Every IntelliJ Platform action should override `AnAction.update()` and must over
   An object of the [`AnActionEvent`](%gh-ic%/platform/editor-ui-api/src/com/intellij/openapi/actionSystem/AnActionEvent.java) type is passed to this method and contains information about the current context for the action.
   Actions are made available by changing state in the [`Presentation`](%gh-ic%/platform/editor-ui-api/src/com/intellij/openapi/actionSystem/Presentation.java) object associated with the event context.
   As explained in [Overriding the `AnAction.update()`  Method](#overriding-the-anactionupdate-method), it is vital `update()` methods _execute quickly_ and return execution to platform.
+* `AnAction.getActionUpdateThread()` return an [`ActionUpdateThread`](%gh-ic%/platform/editor-ui-api/src/com/intellij/openapi/actionSystem/ActionUpdateThread.java),
```

Comment:
```suggestion
* `AnAction.getActionUpdateThread()` (2022.3+) return an [`ActionUpdateThread`](%gh-ic%/platform/editor-ui-api/src/com/intellij/openapi/actionSystem/ActionUpdateThread.java),
```
I would add information about the version it is available in so that developers targeting older versions are not confused that they can't find it if they try straight away. The sentence at the end is about DevKit inspection. There is info on `working_with_custom_actions.md`, but I think it should be in both places.